README.md

```diff
@@ -1 +1,283 @@
-# backend
+
+# hot-coffee-sql
```

Comment:
maybe give it more entertaining name like `frappuccino` or `cold-coffee` or `hottest-coffe`